docs/setup/install-new.md

```diff
@@ -162,6 +162,7 @@ az aks get-credentials --resource-group $resourceGroupName --name $aksClusterNam
   ```bash
   kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.5.5/deploy/infra/deployment.yaml
```

Comment:
we need to change 1.5.5 to 1.6.0. 1.5.5 will not work anymore.

Reply:
Updated